server/public/index.html

```diff
@@ -4,15 +4,15 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <link rel="stylesheet" type="text/css" href="<%= BASE_URL %>css/styles.css">
+    <link rel="icon" href="/favicon.ico">
+    <link rel="stylesheet" type="text/css" href="/css/styles.css">
     <title>Vue</title>
-  </head>
+  <link href="/app.js" rel="preload" as="script"></head>
```

Comment:
You've got a hanging </head> tag. I think it should be on the next line.

Reply:
I committed this because it was generated by `npm run build-dev`. I could fix this, but since this is autogenerated I don't even think it should be in source control. ¯\_(ツ)_/¯

Reply:
Hmm, that's odd. Because that pre-load wasn't there. Also this isn't fully auto-generated, as this is where we'd put CDN links and links to other style sheets. So I don't think it gets fully overwritten upon `npm run build-dev`.

Reply:
Resolved. I have changed my mind. This is okay. This file is dynamically changing every time the user interacts with anything here because of Vue, so a few misplaced closing tags is not a deal.